agr_java_core/src/main/java/org/alliancegenome/cache/CacheAlliance.java

```diff
@@ -23,6 +24,7 @@
     //GENE_DISEASE_ANNOTATION(),
 
     ALLELE_GENE(Allele.class, (3944 * 100) + 741634), // min: 607 max: 741634 mean: 3944
+    ALLELE_VARIANT_SEQUENCE_GENE(AlleleVariantSequence.class, (3944 * 100) + 741634), // min: 607 max: 741634 mean: 3944
```

Comment:
@cmpich Probably should run some stats on these "documents" in order to size this correctly.

Reply:
Yes, I will do this once I am sure it's working otherwise.